optuna_dashboard/_app.py

```diff
@@ -297,14 +297,18 @@ def get_study_detail(study_id: int) -> BottleViewReturn:
             response.status = 404  # Not found
             return {"reason": f"study_id={study_id} is not found"}
         trials = get_trials(storage, study_id)
-        intersection, union, has_intermeridate_values = get_cached_extra_study_property(
-            study_id, trials
-        )
+        (
+            intersection,
+            union,
```

Comment:
[nit] `union_search_space` looks more clear since now we have `union_user_attrs`. Leaving a TODO comment might be good for now to keep the PR small.

```suggestion
            intersection_search_space,
            union_search_space,
```

Reply:
Sure!

Reply:
```suggestion
            # TODO: intersection_search_space and union_search_space look more clear since now we have union_user_attrs.
            intersection,
            union,
```